content/post/15-cutting-out-the-noise/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Cutting Out the Noise"
+date: 2011-12-26T00:00:00-07:00
+draft: false
+categories:
+- entrepreneurship
+- Zen
```

Comment:
```suggestion
- zen
```

Reply:
Thanks for reviewing, I still have bunch of blogs left. I will match my current directory with the above picture.